ls1intum/Artemis
# Fix issue with missing result for programming exercises in exam mode after refreshing the browser

Fixes #2779 which was introduced by unintended changes in #2738
Also improve REST URLs to follow the standard and to distinguish those better

### Checklist

### Steps for Testing
1. Create an exam and change the exercise group order
2. Participate as student in an exam and double check that everything is still working (some REST Urls were changed)
3. Submit code for one programming exercise and wait until the results appears
4. Reload the page
5. Double check that the result is displayed correctly
6. Also try out a test run and make sure nothing is broken here
